ethsigner/core/src/integration-test/java/tech/pegasys/ethsigner/jsonrpcproxy/model/jsonrpc/TransactionBuilder.java

```diff
@@ -12,6 +12,8 @@
  */
 package tech.pegasys.ethsigner.jsonrpcproxy.model.jsonrpc;
 
+import io.vertx.core.json.JsonObject;
+
```

Comment:
I am sure @rain-on will argue about Jackson's Json vs. vertx JsonObject :)

Reply:
Could use JsonNode from Jackson or plain old Map I guess, but we are already using JsonObject extensively in EthSigner. I agree it would be nice to be less dependent on vertx